server/src/main/java/com/ncc/neon/services/ClusterService.java

```diff
@@ -211,6 +260,66 @@ private void aggregateNumbersInNewData(String fieldNameKey, String aggregationNa
         }
     }
 
+    /**
+     * Converts the incoming text bin e.g. "a", "bd" into a number.
+     *
+     * @param text the bin to be converted
+     * @return a BigDecimal corresponding to the index of the bin
+     */
+    private BigDecimal convertTextBinToNumber(String text) {
+        final String ALPHABET = "abcdefghijklmnopqrstuvwxyz";
+        final int ALPHABET_LENGTH = 26;
```

Comment:
Would ALPHABET and ALPHABET_LENGTH be better scoped as static class variables?

Reply:
Fixed.